### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

Update readmes now the `cp-feature-refactor` process is complete!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-2. Perform segmentation
-3. Define a new single-cell focused dataset of isolated single cells
-4. Compute single cell embeddings
+0. Download the Cell Health Cell Painting dataset ([IDR0080](https://idr.openmicroscopy.org/webclient/?show=screen-2701))
+1. Perform nuclei segmentation
+2. Extract single cell embeddings
     - Using CellProfiler
     - Using DeepProfiler
-5. Process both kinds of embeddings using pycytominer
+3. Prerocess both kinds of embeddings using pycytominer
```
Comment (MikeLippincott) [inline README.md]:
```suggestion
3. Preprocess both kinds of embeddings using pycytominer
```

File: README.md [hunk 2]
```diff
@@ -19,15 +20,7 @@ These data were originally used as part of the publication [Way et al. 2021](htt
...
-
-```bash
-# Make sure to follow all steps in `0.data-download/README.md` first
-# This includes one step to test and confirm download configuration
-cd 0.image-download
-bash download_images.sh
-```
-
-## Step 2 - Segmentation
+The [Way Lab](https://www.waysciencelab.com/) always strives for readable, reproducible computational biology analyses and workflows. If you struggle to understand or reproduce anything in this repository please [file an issue](https://github.com/WayScience/cell-health-data/issues/new)!
```
Comment (MikeLippincott) [inline README.md:26]: I like this statement making things transparent

File: README.md (hunk 1 above)
Comment (MikeLippincott) [reply README.md]: or should it be `process`?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
+## Overview
 This is a data repository storing instructions on how to:
 
-1. Download the Cell Health Cell Painting dataset ([IDR0080](https://idr.openmicroscopy.org/webclient/?show=screen-2701))
-2. Perform segmentation
-3. Define a new single-cell focused dataset of isolated single cells
-4. Compute single cell embeddings
+0. Download the Cell Health Cell Painting dataset ([IDR0080](https://idr.openmicroscopy.org/webclient/?show=screen-2701))
+1. Perform nuclei segmentation
```
Comment (MikeLippincott) [inline README.md]:
```suggestion
1. Perform single nuclei segmentation
```

**Review by MikeLippincott — APPROVED** [review]
A few spelling things I commented on might need to change, but LGTM.

File: README.md (hunk 1 above)
Comment (roshankern) [reply README.md]: It should be `Preprocess`. Thanks for catching the typo!

**Pull request merged**